### ISSUE TITLE
Marketing Tool Item Images need alt tags

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75440]
### Quick summary

On Marketing/Tools, each tool item has an image that needs either alt text or an empty alt text tag. Alt text should be used if the image is important to the context, and an empty tag should be used if the image is not important to the context and can be skipped by screen readers.

[Here's a decision three from W3C to help determine if text or an empty tag should be used.](https://www.w3.org/WAI/tutorials/images/decision-tree/#:~:text=wasn't%20there%3F-,Yes%3A,the%20link%20or%20action%20taken)

### Steps to reproduce

1. Start on marketing/tools
2. Inspect the page to look at the images on the page.
3. See that the images on the tool item bodies do not have alt tags. 
`<img class="tools__feature-list-item-body-image" src="/calypso/images/../images/built-by-wp-vert-blue-bf70671b9a93e895dd18.png">`

### What you expected to happen

All images should have an alt tag. If alt text is unnecessary, the alt tag should be present but empty.

### What actually happened

None of the tools have alt text associated with their images.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple

**Comment by davemart-in** [comment]
PR: https://github.com/Automattic/wp-calypso/pull/76955

**Comment by davemart-in** [comment]
Should be fixed now.
